google/auth/compute_engine/credentials.py

```diff
@@ -84,7 +83,7 @@ def __init__(
         self._scopes = scopes
         self._default_scopes = default_scopes
         self._universe_domain_cached = False
-        self._universe_domain_request = google_auth_requests.Request()
+        self._universe_domain_request = None
```

Comment:
nit: Consider exposing the request as an optional parameter.

Reply:
Removed the universe_domain_request completely per @arithmetic1728 's suggestion